core/src/main/scala/cats/package.scala

```diff
@@ -70,7 +70,12 @@ package object cats {
 
   type Endo[A] = A => A
 
-  val catsInstancesForId: Bimonad[Id] with CommutativeMonad[Id] with NonEmptyTraverse[Id] with Distributive[Id] =
+  @deprecated("retained for binary compatibility", "2.7.1")
+  private[cats] def catsInstancesForId
+    : Bimonad[Id] with CommutativeMonad[Id] with Comonad[Id] with NonEmptyTraverse[Id] with Distributive[Id] =
+    catsInstancesForIdBinCompat1
+  val catsInstancesForIdBinCompat1
+    : Bimonad[Id] with CommutativeMonad[Id] with NonEmptyTraverse[Id] with Distributive[Id] =
```

Comment:
Why do we need this change? 🤔 - I thought only the forwarder in `Invariant` needs a change